marklogic-data-hub-central/ui/e2e/cypress/integration/monitor/monitorCustomDatePicker.spec.tsx

```diff
@@ -54,17 +54,22 @@ describe("Monitor Tile", () => {
     monitorSidebar.getStartTimeSelect().click();
     monitorSidebar.getStartTimeSelectOption("Custom").click();
     monitorSidebar.getDateRangePicker().should("be.visible").click();
-    monitorSidebar.getTodayItemInDateRangePicker().click();
+    monitorSidebar.getTodayItemInDateRangePicker().click({force: true, multiple: true});
     monitorSidebar.getAllAvailableDaysInDateRangePicker().then($listOfTd => {
-      let found: boolean = false;
-      cy.wrap($listOfTd).each($td => {
-        if (found) {
-          cy.wrap($td).click();
-          found = false;
-        }
-        if ($td.hasClass("today")) {
-          found = true;
-        }
+      //TODO: investigate cypress dom re-render in DHFPROD-8671
```

Comment:
are you sure this is the issue? I think we're trying to do twice the same thing. I suggest removing these lines, if there's a case where we need this please let me know